04-store-api/starter/populate.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+
+const connectDB = require('./db/connect');
+const Product = require('./models/product');
+
+const jsonProducts = require('./products.json');
+
+const start = async () => {
+  try {
+    await connectDB(process.env.MONGODB_URI);
+    await Product.deleteMany()
+    await Product.create(jsonProducts)
```

Comment:
😮 that's a bit intense! Be careful about naming scripts that take destructive actions; I'd definitely suggest calling this `destroy_and_repopulate_db.js`.

Also, do we need to delete all products? What is the reasoning behind this design decision & can we think of a better way to have a data population script without needing to wipe out pre-existing data?